tests/v2_validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -93,7 +93,7 @@
 RANCHER_DNS_SERVER = "169.254.169.250"
 RANCHER_DNS_SEARCH = "rancher.internal"
 RANCHER_FQDN = "rancher.internal"
-SERVICE_WAIT_TIMEOUT = 120
+SERVICE_WAIT_TIMEOUT = int(os.environ.get('SERVICE_WAIT_TIMEOUT', "120"))
```

Comment:
@cjellick In the stress testing that i did with @moelsayed, we used to deploy several stacks and then force upgrade ipsec and kubernetes, upgrading ipsec is done by `execute_rancher_cli` which uses SERVICE_WAIT_TIMEOUT to wait for the command to be executed on the rancher cli test container, with more stacks deployed the ssh command took longer to be executed so we had to parameterize this variable.